sdk/storage/storage-file-share/test/node/highlevel.node.spec.ts

```diff
@@ -283,7 +284,7 @@ describe("Highlevel Node.js only", () => {
   it("downloadToBuffer should throw an error if the count (size in bytes) is too large", async () => {
     let error;
     try {
-      await fileClient.downloadToBuffer(undefined, 4 * 1024 * 1024 * 1024);
+      await fileClient.downloadToBuffer(undefined, (buffer as any).constants.MAX_LENGTH + 1);
```

Comment:
```suggestion
      // casting to "any" is required since @types/node@8 doesn't have constants though it is present, "as any" can be removed once we move from @types/node v8 to v10
      await fileClient.downloadToBuffer(undefined, (buffer as any).constants.MAX_LENGTH + 1);
```